build.gradle

```diff
@@ -27,6 +27,7 @@ allprojects {
             force 'com.android.support.constraint:constraint-layout:1.1.3'
             force 'io.reactivex.rxjava2:rxjava:2.2.7'
             force 'io.reactivex.rxjava2:rxandroid:2.1.1'
+            force 'io.mockk:mockk:1.9.3.kotlin12'
```

Comment:
We use Kotlin 1.30, not 1.20, so use 1.9.3 version without suffix.
For multiplatform use "io.mockk:mockk-common:{version}".
Does it work on native platform? Does't seem so, as I can see only jvm/js is supported. Probably we need to use stubs.